Makefile

```diff
@@ -30,8 +30,12 @@ format: node_modules
 lint: node_modules
 	npm run lint
 
+.PHONY: test
```

Comment:
Revert changes to Makefile

Reply:
Are you sure as if I did that user wouldn't see test failures, example of the output:
```shell
 PASS  __tests__/buf-setup.test.ts
  buf-setup
    go-version-file
      ✓ return version if go_version_file isn't set (122 ms)
      ✓ parses go.mod format if go_version_file contains go.mod basename (106 ms)
      ✓ returns contents if go_version_file is set to non go.mod filename (107 ms)
      ✓ reports failure if go_version_file doesn't exist (2 ms)

------------|---------|----------|---------|---------|-----------------------------------------
File        | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                       
------------|---------|----------|---------|---------|-----------------------------------------
All files   |   48.96 |    26.82 |    62.5 |   48.96 |                                         
 buf.ts     |   12.85 |        0 |       0 |   12.85 | 31-186                                  
 error.ts   |     100 |      100 |     100 |     100 |                                         
 run.ts     |      75 |    33.33 |     100 |      75 | 28,39,48,63,69,75,87-96,100-103,107-110 
 version.ts |     100 |    85.71 |     100 |     100 | 78                                      
------------|---------|----------|---------|---------|-----------------------------------------
Test Suites: 1 passed, 1 total
Tests:       4 passed, 4 total
Snapshots:   0 total
Time:        1.693 s, estimated 3 s
Ran all test suites.
```